scripts/ASTImportTest.sh

```diff
@@ -98,7 +98,7 @@ do
         NSOURCES=$((NSOURCES - 1))
         for i in $OUTPUT;
         do
-            testImportExportEquivalence $i $OUTPUT
+            testImportExportEquivalence "$i" "$OUTPUT"
```

Comment:
If `$OUTPUT` is not quoted, `testImportExportEquivalence()` gets source names as separate arguments. But the function assumes that its second argument is a big string consisting of concatenated names.

This means that we were never really testing the equivalence.